<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>381</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Профсоюз.Льготы</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background: rgb(240, 240, 240)</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>381</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>36</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(254, 127, 94);
color: 'white'</string>
   </property>
   <property name="text">
    <string>     Профсоюз.Льготы</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(254, 127, 94)</string>
   </property>
   <property name="text">
    <string>ФИО:</string>
   </property>
  </widget>
  <widget class="QLabel" name="fio_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>355</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(254, 127, 94)</string>
   </property>
   <property name="text">
    <string>Дата рождения:</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_born_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string>Неизвестно</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(254, 127, 94)</string>
   </property>
   <property name="text">
    <string>Дата вступления в профсоюз:</string>
   </property>
  </widget>
  <widget class="QLabel" name="date_enter_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>181</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string>Неизвестно</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(254, 127, 94)</string>
   </property>
   <property name="text">
    <string>Положение в профсоюзе:</string>
   </property>
  </widget>
  <widget class="QLabel" name="pol_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>360</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string>Неизвестно</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>400</y>
     <width>231</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(254, 127, 94)</string>
   </property>
   <property name="text">
    <string>Должность:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>-16</x>
     <y>510</y>
     <width>421</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: 'white';
background: 'white';
border: 2px solid 'black'</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="qr_button">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>470</y>
     <width>100</width>
     <height>120</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px;
background: 'white'</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>qr.png</normaloff>qr.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>130</width>
     <height>121</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="home_btn">
   <property name="geometry">
    <rect>
     <x>25</x>
     <y>520</y>
     <width>71</width>
     <height>71</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>24</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: 'white';
border: 1px;
color: 'white'</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>home.PNG</normaloff>home.PNG</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="ur_help_btn">
   <property name="geometry">
    <rect>
     <x>286</x>
     <y>520</y>
     <width>70</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border: 1px;
background: 'white'</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>help.png</normaloff>help.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>64</width>
     <height>64</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="dolj_lab">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>440</y>
     <width>361</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background: rgb(240, 240, 240);
color: rgb(70, 70, 70)</string>
   </property>
   <property name="text">
    <string>Неизвестно</string>
   </property>
  </widget>
  <widget class="QPushButton" name="change_date_btn">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>200</y>
     <width>141</width>
     <height>32</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgb(254, 127, 94);
color: 'white';
border: 2px solid rgb(254, 127, 94);
border-radius: 15px
}

QPushButton:pressed {
background: rgb(234, 107, 74)
}</string>
   </property>
   <property name="text">
    <string>Изменить</string>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_btn">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>471</y>
     <width>113</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Gill Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {
background: rgb(254, 127, 94);
color: 'white';
border: 2px solid rgb(254, 127, 94);
border-radius: 15px
}

QPushButton:pressed {
background: rgb(234, 107, 74)
}</string>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
